### DISCUSSION TITLE
Add missing docs for `compoundVariants` prop behavior

### DISCUSSION THREAD

**Discussion by Just-Moh-it** [joe-bell/cva#63] [Q&A] [answered]
Having looked through the README and everywhere else in this project, I wasn't able to find what `compoundVariants` does or how it can be used. Having never used stitches, I had no knowledge about it and my understanding was the same as that of @james-elicx

> ... The compoundVariants property should allow an object to be provided to it. The object can contain keys that each map to a combination of variants. These keys can then be used on components using a compoundVariant="primary" prop ...
> ~ @james-elicx

(source: issue #37)

> Yeah I think maybe there's some misunderstanding here: compound variants should only apply if two or more variant conditions are met...
> ~ @joe-bell

After digging through the issues, I found the above quote, and then things snapped into place. Though I'm still confused and would have to move forward with hit-and-trial to know how this works.

I could easily imagine new users being super confused about this prop's behavior since it's not in the docs (and if it is, it's hard to find)

Would be great if info about the prop is added to the docs (ideally with examples).

**Comment by joe-bell** [comment, marked as answer]
Shipped in #76
